### DISCUSSION TITLE
`Mage` modules you not use?

### DISCUSSION THREAD

**Discussion by sreichel** [OpenMage/magento-lts#4140] [Ideas]
We have removed `Mage_Poll` for example, but I think there are others that are not used very often and could be optional.

- Mage_Captcha
- ~~Mage_GoogleAnalytics~~
- Mage_Tag
- ~~Mage_Rating~~
- ~~Mage_Rewiew~~
- Mage_Rss
- [Mage_Sendfriend](https://github.com/OpenMage/magento-lts/discussions/4186)
- ...

**Comment by addison74** [comment]
Although I agree, there are some aspects that should be discussed before certain modules are removed from the distribution package:

- Those who are interested in testing OpenMage for the first time should get a full featured e-commerce solution, but they should be informed that there are separate modules. Unfortunately, these modules are not installed with a click from the Backend, as in the case of other competitors, and they are not listed in the Backend.

- Removing these modules means that we will no longer have the possibility of feedback to improve or solve issues.

On the other hand, modularization has its advantages, so that each module can be maintained separately.

**Comment by alexh-swdev** [comment]
For example GA and Rating/Review are in use

**Comment by colinmollenhour** [comment]
In my opinion, RSS is mostly dead and is a good candidate for moving to a separate module.

I've never used Tag but honestly never really looked at it closely... But it seems like a good one to remove if there are no serious objections.

The Captcha seems too dated.. But, do other modules that provide ReCaptcha piggy-back off of it?

You saw my discussion on Sendfriend already. :)

**Comment by sreichel** [comment]
How about ...

- move every module to its own repo
- sync it via github actions from this repo, or use https://github.com/symplify/monorepo-builder

See

- https://github.com/zf1s/zf1
- https://github.com/zf1s/zf1/blob/master/composer.json

I really like that idea, so everybody can easily add/remove modules he does/does not need.

IMHO it would be a great addition for the future.

It requires some cleanup, thats one reason i started https://github.com/OpenMage/magento-lts/pull/4262

**Comment by sreichel** [comment]
WIP ... 

- https://github.com/OpenMage/magento-lts/pull/4618
